.github/label-actions.yml

```diff
@@ -0,0 +1,12 @@
+# Configuration for Label Actions - https://github.com/dessant/label-actions
+
+# The `enhancement` label is added to issues
+enhancement:
+  # Post a comment
+  comment: >
+    Thank you for submitting a feature request. Here’s what to expect next. 
+    Your proposal is open and is now being [triaged](https://github.com/orgs/carbon-design-system/projects/51/views/1) by the Carbon team. 
```

Comment:
Should this also add the `proposal: open` label?

Reply:
Oo I like that - updating now!

Reply:
hmm I actually don't see an immediate way to do that - but this PR is just a test PR - so that's something we can play with